pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -110,11 +111,15 @@ func (r ResourceLogsUnmarshaler) UnmarshalLogs(buf []byte) (plog.Logs, error) {
 			nanos, err := getTimestamp(log)
 			if err != nil {
 				r.Logger.Warn("Unable to convert timestamp from log", zap.String("timestamp", log.Time))
-				continue
 			}
 
 			lr := logRecords.AppendEmpty()
```

Comment:
nit: you can move `lr := logRecords.AppendEmpty()` to before you call `getTimestamp`, and then make the check below in an `} else {` block.